docs/build/getting-started/networks/overview.md

```diff
@@ -1,5 +1,148 @@
+| [Storage deposits](#storage-deposits)                          | None                                                                         | Yes                                                                                                                                                                                                                                                                                                                                                                                   |
+| [Output unlock conditions](#output-unlock-conditions)          | [Address](https://wiki.iota.org/learn/stardust/core-concepts/output-unlock-conditions) | [Address](https://wiki.iota.org/learn/stardust/core-concepts/output-unlock-conditions), [Storage Deposit Returns](https://wiki.iota.org/learn/stardust/core-concepts/output-unlock-conditions#storage-deposit-return), [Expiration](https://wiki.iota.org/learn/stardust/core-concepts/output-unlock-conditions#expiration), [Timelock](https://wiki.iota.org/learn/stardust/core-concepts/output-unlock-conditions#timelock) |
+| [IOTA Smart Contract (ISC)](#iota-smart-contracts-isc) Support | No                                                                           | In alpha state                                                                                                                                                                                                                                                                                                                                                                        |
+
+## Stability
+
+Shimmer is designed as a staging network to test features publicly before deploying them on the IOTA network, so it
+will always have more features than IOTA. As this may come at the cost of some stability, before any protocol changes
+are deployed on the [Shimmer Mainnet](../endpoints/shimmer.md), they have been tested
+in the [Shimmer Testnet](../endpoints/testnet.md).
+
+## Store Data
+
+### IOTA
+
+In IOTA, you can attach
+an [IndexationPayload](https://wiki.iota.org/shimmer/tips/tips/TIP-0006/#indexation-payload)
+to a message or
+any `SignedTransaction`.
+The IndexationPayload allows you to add an index to a data message and arbitrary data.
+
+### Shimmer
+
+#### INX-Poi
+
+You can use the [INX-PoI](https://wiki.iota.org/shimmer/inx-poi/welcome/) node extension to verify if any piece of data
+was included in the Tangle. This feature does not rely on
+the [Chronicle](/maintain/chronicle/welcome) permanode, so it drastically reduces the node operator's storage
+requirements.
+
+#### Tagged Data Payload
+
+You can use a [Tagged Data Payload](https://wiki.iota.org/shimmer/tips/tips/TIP-0023/) to attach binary data and a tag
+to extend an object with arbitrary data.
```

Comment:
```suggestion
to extend an object with arbitrary data. Keep in mind this will not be indexed by the core nodes and you might want to create your own indexer based on INX or find a community driven project
```